components/datagrid/HeaderRowOverlay/theme.css

```diff
@@ -0,0 +1,12 @@
+@import '@teamleader/ui-colors';
+@import '@teamleader/ui-typography';
+
+.header-row-overlay {
+  background: var(--color-neutral-light);
+  position: absolute;
+  left: 48px;
+  right: 0;
+  height: 48px;
+  overflow-x: auto;
+  z-index: 3;
```

Comment:
Any reason why `3`?

Reply:
This is because, for the DataGrid, z-index 1 and 2 are already in use:

```
.has-blend-left .cell:first-child::before,
.has-blend-right .cell:last-child::before {
  z-index: 2;
}
```

```
.has-border-around {
  z-index: 1;
}
```